src/libs/actions/Report.js

```diff
@@ -1449,6 +1450,90 @@ function createPolicyRoom(policyID, reportName, visibility) {
         .finally(() => Onyx.set(ONYXKEYS.IS_LOADING_CREATE_POLICY_ROOM, false));
 }
 
+/**
+ * Add a workspace room optimistically and navigate to it.
+ *
+ * @param {Object} policy
+ * @param {String} reportName
+ * @param {String} visibility
+ */
+function addWorkspaceRoom(policy, reportName, visibility) {
```

Comment:
```suggestion
function addPolicyReport(policy, reportName, visibility) {
```
Can you please update this everywhere? Full context is in this thread: https://expensify.slack.com/archives/C02HWMSMZEC/p1661522786746159

Reply:
Ah right, I should have remembered. I'll update now.